src/components/Header/BurgerButton/BurgerButton.jsx

```diff
@@ -15,8 +15,6 @@ const BurgerButton = ({ opened, onClick, ...rest }) => (
     $active={opened}
     {...rest}
   >
```

Comment:
Il faudrait traduire le` aria-label` plus haut
```suggestion
<Button
    aria-label={
      opened ? 'Fermer le menu de navigation' : 'Ouvrir le menu de navigation'
    }
```